game/static/game/css/board.css

```
@@ -2521,3 +2521,21 @@ body.blindfold-mode .capture-ring {
 }
 
+/* ================= PUZZLE HINT SYSTEM ================= */
+
+.puzzle-hint {
+    position: relative;
+    box-shadow: inset 0 0 0 4px #f0c040;
+    animation: puzzleHintPulse 1.2s infinite;
+    z-index: 3;
+}
+
+@keyframes puzzleHintPulse {
```

Stylelint (17.12.0)

[error] Expected keyframe name "puzzleHintPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)